feature/platform/tests/optics_power_and_bias_current_test/optics_power_and_bias_current_test.go

```diff
@@ -179,28 +165,19 @@ func TestOpticsPowerUpdate(t *testing.T) {
 	}
 }
 
-// findTransceiverName provides name of transciever port corresponding to interface name
+// findTransceiverName provides name of transceiver port corresponding to interface name
 func findTransceiverName(dut *ondatra.DUTDevice, interfaceName string) (string, error) {
-	var (
-		transceiverMap = map[ondatra.Vendor]string{
-			ondatra.ARISTA:  " transceiver",
-			ondatra.CISCO:   "",
-			ondatra.JUNIPER: "",
-			ondatra.NOKIA:   "-transceiver",
-		}
-	)
-	transceiverName := interfaceName
-	name, ok := transceiverMap[dut.Vendor()]
-	if !ok {
-		return "", fmt.Errorf("No transceiver interface available for DUT vendor %v", dut.Vendor())
-	}
-	if name != "" {
+	switch dut.Vendor() {
```

Comment:
@LimeHat , please could you add transceiver name for JUNIPER.  Here is the code for the same.

case ondatra.JUNIPER:
		interfaceSplit := strings.Split(interfaceName, "/")
		interfaceSplit1 := strings.Split(interfaceSplit[0], "-")
		transceiverName := "FPC" + interfaceSplit1[1] + ":PIC" + interfaceSplit[1] + ":PORT" + interfaceSplit[2] + ":Xcvr0"
		return transceiverName, nil